library/datasort.h

```diff
@@ -81,8 +81,10 @@ struct datasort_cfg {
 	struct list_head		sorted_chunks;
 	/* Result of mergesort */
 	struct datasort_chunk		*result;
-	/* Datasort directory */
-	char				*dir;
+	/* Datasort directory - chunks are merged into single sorted blob here */
+	char				*datasort_dir;
+	/* Chunks directory - if not defined, then chunks are stored in datasort_dir */
+	char				*chunks_dir;
```

Comment:
Main purpose of separation of one datasort-dir into multiple directories is achieving optimal i/o performance of overall datasort using multiple disk devices.
1. read blob from disk1 and write its chunks into disk2
2. sort every chunk in disk2: 1 read + 1 write operation on disk2
3. merge chunks from disk into sorted blob to disk1

But, if merge process will store sorted blob to disk2, then moving this sorted blob will lead to copying it from disk2 to disk1. Thereby additional i/o operations on disk2 arises: 1 extra read & write on disk2.